go-app-ussd_higherhealth_healthcheck.js

```diff
@@ -1701,32 +1702,81 @@ go.app = (function () {
         return self.states.create(name, opts);
       });
     });
-
-    self.states.add("state_welcome", function(name) {
+    self.states.add("state_welcome", function(name, opts) {
       self.im.user.answers.google_session_token = crypto.randomBytes(20).toString("hex");
       var question;
-      if (self.im.user.answers.returning_user) {
+      var date = moment().format('YYYY-MM-DD');
+      var msisdn = self.im.user.addr;
+      return new JsonApi(self.im).get(
+        // use dictionary params
+        self.im.config.eventstore.url + "/api/v3/covid19triage/", {
+        headers: {
+          "Authorization": ["Token " + self.im.config.eventstore.token],
+          "User-Agent": ["Jsbox/HH-Covid19-Triage-USSD"]
+        },
+        params: {
+          "timestamp_gt": date + "T00:00:00+02:00",
+          "msisdn": msisdn
```

Comment:
Do we need to make sure that this msisdn is normalised? eg `+27820001001` instead of `27820001001`?